plugins/rapid7_insightcloudsec/help.md

```diff
@@ -421,12 +494,16 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 2.1.0 - Updated SDK to the latest version | Adding a get resource id functionality
 * 2.0.0 - Enable plugin to run in cloud | Remove actions using defective API endpoints
 * 1.0.0 - Initial plugin | Add Get Account Details, Get Resource Details, List Resource Tags, Create Exemption, Remove Exemption, Detach Policy, Run Bot on Demand and Switch Organization actions
 
 # Links
 
+* [Rapid7 InsightCloudSec](https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/rapid7_insightcloudsec)
+* [Docs](https://docs.rapid7.com/insightcloudsec/api/v2/)
+
 ## References
 
 * [Rapid7 InsightCloudSec](https://www.rapid7.com/products/insightcloudsec/)
-
+* You can find testing credentials under the Shared-Integration Lab/DivvyCloud - Demo @ 1Password
```

Comment:
Imo this should be removed

Reply:
@dlaverty-r7 asked me to add information about where the Api-Key comes from for testing, as the name DivvyCloud may not be obvious. Dympna, can you confirm if it's okay?

Reply:
Ah sorry, @lcwiklinski-r7 I should have clarified, I meant can you add a comment in our 1password creds store? This comment can be removed from the help.md as all of this code is visible by customers.

Reply:
ok, done 👍